src/SourceGenerators/Uno.UI.SourceGenerators/BindableTypeProviders/DependencyObjectAvailabilityGenerator.cs

```diff
@@ -91,26 +96,28 @@ internal void Generate(GeneratorExecutionContext context)
 						var bindableTypes = from module in modules
 											from type in module.GlobalNamespace.GetNamespaceTypes()
 											where (
-												(
-													type.GetAllInterfaces().Any(i => SymbolEqualityComparer.Default.Equals(i, _dependencyObjectSymbol))
-												)
+												type.GetAllInterfaces().Any(i => SymbolEqualityComparer.Default.Equals(i, _dependencyObjectSymbol))
```

Comment:
```suggestion
												type.AllInterfaces.Any(i => SymbolEqualityComparer.Default.Equals(i, _dependencyObjectSymbol))
```

Reply:
`DependencyObject` is an interface, so it may be filtered out as well, and we need it here.